cmd/registry/cmd/upload-manifest.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"log"
+	"context"
+	"github.com/spf13/cobra"
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/cmd/registry/controller"
+	"github.com/apigee/registry/rpc"
+	"google.golang.org/protobuf/proto"
+	"github.com/apigee/registry/cmd/registry/core"
+
+)
+
+func init() {
+	uploadCmd.AddCommand(uploadManifestCmd)
+	uploadManifestCmd.Flags().String("project", "", "ProjectID this manifest should be associated with.")
```

Comment:
For consistency across the registry command, please use `project` when a full project resource name is required (`projects/PROJECT_ID`) and `project_id` when we are only expecting the project ID.

Reply:
It seems like `project_id` is the more common pattern right now, but it seems rare that we would ever refer to an API, Version, Spec, or Artifact by its ID alone. That said, maybe we should lean toward using full resource names.

Reply:
Done, I changed this to `project_id` to be consistent.